test/Microsoft.IdentityModel.Tests/KeyingMaterial.cs

```diff
@@ -420,14 +433,14 @@ static KeyingMaterial()
 
         }
 
-#if NET452 || NET45
+#if NET451 || NET452 || NET45
```

Comment:
I think we only need NET452

Reply:
I think we also need NET45, but I don't think NET451 is necessary.